language-support/codegen-common/src/test/scala/com/digitalasset/daml/codegen/UtilSpec.scala

```diff
@@ -42,7 +42,7 @@ object UtilSpec {
 
   val trivialEnvInterfaceGen: Gen[EnvironmentInterface] = {
     val fooRec = Record(ImmArraySeq.empty)
-    val fooTmpl = InterfaceType.Template(fooRec, DefTemplate(Map.empty, Map.empty, None))
+    val fooTmpl = InterfaceType.Template(fooRec, DefTemplate(Map.empty, Map.empty, None, Seq.empty))
```

Comment:
Do you setup something somewhere where a template does implement interfaces? (just checking)

Reply:
The relevant tests are only in the integration tests, because the code needs to be both compile-able & semantically correct which I can only check via running it against a ledger.

Reply:
Ok. Here the thought (during my review) was just that maybe you want to run a test where the DefTemplate implements interfaces. It's very minor.